integration/discovery/discovery_test.go

```diff
@@ -634,6 +654,97 @@ var _ = Describe("DiscoveryService", func() {
 		Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit(1))
 		Expect(sess.Err).To(gbytes.Say(`access denied`))
 	})
+
+	When("a peer is bootstrapped from a snapshot", func() {
```

Comment:
Is it not possible to have this test at the end of the existing `It`, as you try to reduce the setup overheads at other places?

Reply:
Moved the snapshot test to an existing `It`.